hnn_core/tests/test_parallel_backends.py

```diff
@@ -142,7 +140,7 @@ def test_run_mpibackend_oversubscribed(self, run_hnn_core_fixture):
         oversubscribed = round(cpu_count() * 1.5)
         with MPIBackend(n_procs=oversubscribed) as backend:
             assert backend.n_procs == oversubscribed
-            simulate_dipole(net)
+            simulate_dipole(net, tstop=40)
 
```

Comment:
I'm guessing somewhere around here would be a good place to add Assertion tests for invalid `tstop` inputs.

Reply:
I'm not exactly sure I follow? Currently, the checks are inside `simulate_dipole`

Reply:
MIssed them! I was too hasty with scrolling through the (25!) files